setup_namespace

```diff
@@ -6,6 +6,9 @@ if [ "${1-}" = --second-stage ]; then
 	shift
 	mount -t tmpfs -o size=4G tmpfs /tmp
 	"$@"
+	if [ -d /builder/.build ]; then
```

Comment:
This point in the code will not be reached if the prior `$@` returns with non zero exit code due to `set -e`. Maybe better to either define a cleanup permissions function and set a trap on `EXIT`.